[vxl:x64-osx] build failure

**Host Environment**
- OS: MacOS

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install vxl`

**Failure logs** 
`Starting package 12/12: vxl:x64-osx`
`Building package vxl[core]:x64-osx...`
`Could not locate cached archive:` `/Users/catherine/.cache/vcpkg/archives/c7/c78021fec5637ccdf298539ec08aca263050810c.zip`
`-- Downloading https://github.com/vxl/vxl/archive/dac1c7ed8e183e9c6de8b928c8b0294a7bd1d8ee.tar.gz...`
`-- Extracting source /Users/catherine/vcpkg/downloads/vxl-vxl-dac1c7ed8e183e9c6de8b928c8b0294a7bd1d8ee.tar.gz`
`-- Applying patch fix_dependency.patch`
`CMake Error at scripts/cmake/vcpkg_apply_patches.cmake:51 (message):`
`  Applying patch failed.  Checking patch core/CMakeLists.txt...`

`  error: while searching for:`

  

  

`  # coordinate systems`

`  if(BUILD_CORE_GEOMETRY AND BUILD_CORE_NUMERICS)`

`    add_subdirectory(vcsl)`
`    add_subdirectory(vpgl)`

`  endif()`

  

`  error: patch failed: core/CMakeLists.txt:100`

`  error: core/CMakeLists.txt: patch does not apply`

`Call Stack (most recent call first):`
`  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:141 (vcpkg_apply_patches)`
`  scripts/cmake/vcpkg_from_github.cmake:139 (vcpkg_extract_source_archive_ex)`
`  ports/vxl/portfile.cmake:10 (vcpkg_from_github)`
`  scripts/ports.cmake:79 (include)`


`Error: Building package vxl:x64-osx failed with: BUILD_FAILED`

`  Package: vxl:x64-osx`
`  Vcpkg version: 2020.06.15-unknownhash`

**Additional context**
I'm pretty sure my vcpkg is up-to-date.

--- a/ports/vxl/CONTROL
+++ b/ports/vxl/CONTROL
@@ -1,8 +1,7 @@
 Source: vxl
 Version: 2.0.2
-Port-Version: 1
+Port-Version: 2
 Build-Depends: bzip2, expat, libgeotiff, libjpeg-turbo, libpng, shapelib, tiff, zlib
-# Build-Depends: bzip2, dcmtk, expat, libgeotiff, libjpeg-turbo, openjpeg, libpng, shapelib, tiff, zlib
 Description: A multi-platform collection of C++ software libraries for Computer Vision and Image Understanding.
 
 Feature: core-imaging

--- a/ports/vxl/fix_dependency.patch
+++ b/ports/vxl/fix_dependency.patch
@@ -1,5 +1,5 @@
 diff --git a/core/CMakeLists.txt b/core/CMakeLists.txt
-index 3483f9c..a30abb2 100644
+index 3483f9c..57e149b 100644
 --- a/core/CMakeLists.txt
 +++ b/core/CMakeLists.txt
 @@ -100,7 +100,7 @@ endif()
@@ -7,7 +7,7 @@ index 3483f9c..a30abb2 100644
  
  # coordinate systems
 -if(VXL_BUILD_CORE_GEOMETRY AND VXL_BUILD_CORE_NUMERICS)
-+if(VXL_BUILD_CORE_GEOMETRY AND VXL_BUILD_CORE_NUMERICS AND BUILD_CORE_IMAGING)
++if(VXL_BUILD_CORE_GEOMETRY AND VXL_BUILD_CORE_NUMERICS AND VXL_BUILD_CORE_IMAGING)
    add_subdirectory(vcsl)
    add_subdirectory(vpgl)
  endif()

--- a/ports/vxl/fix_dependency.patch
+++ b/ports/vxl/fix_dependency.patch
@@ -1,13 +1,13 @@
 diff --git a/core/CMakeLists.txt b/core/CMakeLists.txt
-index b6af7c2..7ca75d3 100644
+index 3483f9c..a30abb2 100644
 --- a/core/CMakeLists.txt
 +++ b/core/CMakeLists.txt
 @@ -100,7 +100,7 @@ endif()
  
  
  # coordinate systems
--if(BUILD_CORE_GEOMETRY AND BUILD_CORE_NUMERICS)
-+if(BUILD_CORE_GEOMETRY AND BUILD_CORE_NUMERICS AND BUILD_CORE_IMAGING)
+-if(VXL_BUILD_CORE_GEOMETRY AND VXL_BUILD_CORE_NUMERICS)
++if(VXL_BUILD_CORE_GEOMETRY AND VXL_BUILD_CORE_NUMERICS AND BUILD_CORE_IMAGING)
    add_subdirectory(vcsl)
    add_subdirectory(vpgl)
  endif()

--- a/ports/vxl/portfile.cmake
+++ b/ports/vxl/portfile.cmake
@@ -18,7 +18,7 @@ vcpkg_from_github(
 )
 
 set(USE_WIN_WCHAR_T OFF)
-if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+if(VCPKG_TARGET_IS_WINDOWS)
     set(USE_WIN_WCHAR_T ON)
 endif()
 
@@ -54,4 +54,3 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
 file(INSTALL ${SOURCE_PATH}/core/vxl_copyright.h DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
-#

--- a/ports/vxl/portfile.cmake
+++ b/ports/vxl/portfile.cmake
@@ -1,8 +1,8 @@
-set(BUILD_CORE_IMAGING OFF)
+set(VXL_BUILD_CORE_IMAGING OFF)
 if("core-imaging" IN_LIST FEATURES)
-  set(BUILD_CORE_IMAGING ON)
+  set(VXL_BUILD_CORE_IMAGING ON)
   if(EXISTS "${CURRENT_INSTALLED_DIR}/include/openjpeg.h")
-    set(BUILD_CORE_IMAGING OFF)
+    set(VXL_BUILD_CORE_IMAGING OFF)
     message(WARNING "Can't build VXL CORE_IMAGING features with non built-in OpenJpeg. Please remove OpenJpeg, and try install VXL again if you need them.")
   endif()
 endif()
@@ -26,9 +26,9 @@ vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
-        -DBUILD_EXAMPLES=OFF
+        -DVXL_BUILD_EXAMPLES=OFF
         -DBUILD_TESTING=OFF
-        -DBUILD_CORE_IMAGING=${BUILD_CORE_IMAGING}
+        -DVXL_BUILD_CORE_IMAGING=${VXL_BUILD_CORE_IMAGING}
         -DVXL_FORCE_V3P_BZLIB2=OFF
         -DVXL_USING_NATIVE_BZLIB2=TRUE # for disable build built-in bzip2 (v3p/bzlib/CMakeLists.txt#L10-L26)
         -DVXL_FORCE_V3P_CLIPPER=ON # TODO : need add clipper port to turn off
@@ -41,7 +41,7 @@ vcpkg_configure_cmake(
         -DVXL_FORCE_V3P_TIFF=OFF
         -DVXL_FORCE_V3P_ZLIB=OFF
         -DVXL_USE_DCMTK=OFF # TODO : need fix dcmtk support to turn on
-        -DXVL_USE_GEOTIFF=ON
+        -DVXL_USE_GEOTIFF=ON
         -DVXL_USE_WIN_WCHAR_T=${USE_WIN_WCHAR_T}
 )
 

